code/modules/mob/mob_movement.dm

```diff
@@ -519,7 +519,7 @@
 			return FALSE
 		//SKYRAT EDIT ADDITION END
 		m_intent = MOVE_INTENT_RUN
-	plane = (m_intent == MOVE_INTENT_WALK) ? GAME_PLANE_FOV_HIDDEN : GAME_PLANE //SKYRAT EDIT ADDITION
+	plane = ((m_intent == MOVE_INTENT_WALK && !HAS_TRAIT(src, TRAIT_OVERSIZED)) ? GAME_PLANE_FOV_HIDDEN : GAME_PLANE) //SKYRAT EDIT ADDITION - Oversized Overhaul
```

Comment:
We don't need parentheses around the entire ternary expression, this should work just as well. I kept one pair because it'll help a bit with readability, but typically it wouldn't be necessary.
```suggestion
	plane = (m_intent == MOVE_INTENT_WALK && !HAS_TRAIT(src, TRAIT_OVERSIZED)) ? GAME_PLANE_FOV_HIDDEN : GAME_PLANE //SKYRAT EDIT ADDITION - Oversized Overhaul
```